### ISSUE TITLE
H65 Technique is Invalid and Abused to Allow Disappearing Placeholders Over Visible Labels

### ISSUE THREAD

**Issue by pauljadam** [w3c/wcag#100]
I'm seeing many people use this technique as a normative type document to meet the labels and instructions SC that requires visible labels. They are applying it to incorrect situations like many text inputs in a shipping address fieldset with placeholders and no visible labels. I could see how it applies in the 4 specific situations listed in H65 but beyond that it should not be valid. 

This technique causes many developers and designers to say well WCAG allows a title on H65 so all we need is a title or some other visually hidden label. This is a major problem for mobile were everyone loves placeholders over labels. http://pauljadam.com/demos/wcagh65invalid.html

Thank you for any help in fixing the issue or correcting me on it if I'm crazy and visually hidden labels are really allowed on whatever text inputs you want.

**Comment by joshueoconnor** [comment]
The core of this technique is where <label> elements cannot be used. So the technique suggests that alternative is to use the title attribute. 

I guess if people choose to use these techniques as normative, then that is a problem but this technique was (and potentially is) still valid. While not mentioned here, another method could be to use aria-label. The technique isn't exhaustive, or normative. Do you think it could be updated to include aria-label? Would that help?

**Comment by joshueoconnor** [comment]
Any comments?

**Comment by joshueoconnor** [comment]
Surveyed: https://www.w3.org/2002/09/wbs/35422/charter_policy_update_2015/

**Comment by DavidMacDonald** [comment]
I think it allow a title attribute, then it has to display the contents on focus as with jquery tooltip https://jqueryui.com/tooltip/

**Comment by joshueoconnor** [comment]
AWK:  We should update the test procedure to H65.

**Comment by joshueoconnor** [comment]
Title attribute is now better supported, such as with VO.

**Comment by awkawk** [comment]
Preparing to mark as deferred due to inactivity and group resource constraints (https://www.w3.org/WAI/GL/wiki/Handling_Issues).
